fdbclient/MultiVersionTransaction.actor.cpp

```diff
@@ -2312,6 +2312,9 @@ ThreadFuture<Void> MultiVersionApi::updateClusterSharedStateMap(std::string clus
 
 void MultiVersionApi::clearClusterSharedStateMapEntry(std::string clusterFilePath) {
 	MutexHolder holder(lock);
+	TraceEvent("ClearingClusterSharedState").detail("ClusterFilePath", clusterFilePath);
+	auto ssPtr = clusterSharedStateMap[clusterFilePath].get();
```

Comment:
MVC should handle DatabaseSharedState as an opaque structure, otherwise we will have compatibility issues in future versions. One possibility would be to put delRef into the beginning of the structure, and treat the rest of the structure as opaque.